gitbase-playground/templates/statefulset.yaml

```diff
@@ -140,8 +140,14 @@ spec:
               value: "{{ .Values.gitbaseServer.logLevel }}"
             - name: GITBASE_REPOS
               value: "{{ .Values.gitbaseServer.repositoriesDir }}"
+            {{- if ..Values.gitbaseServer.squashEnable }}
```

Comment:
Does it work? Here is 2 dots...

Reply:
It shouldn't... Well spotted! 😅 

@carlosms Can you please send a fix?